scripts/lp-bugs-to-gh-issues.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/python3
+
+from launchpadlib.launchpad import Launchpad
+from github import Github
+
+github_token = Github(os.getenv("GITHUB_TOKEN"))
+github = Github(github_token)
```

Comment:
Think you messed this up a bit.

```suggestion
github = Github(os.getenv("GITHUB_TOKEN"))
```